### DISCUSSION TITLE
Boots to black screen

### DISCUSSION THREAD

**Discussion by immortallyinsane** [Jyers/Marlin#1689] [Q&A] [answered]
Hi, I have an ender 3 v2. Was running 135b 3x3 manual mesh for months and upgraded to 201 3x3 a couple weeks ago. 

Everything was fine until yesterday when I power up the printer and the screen is blank. If I flash it again with 201 3x3 manual mesh it comes back. I thought I was a fluke and changed and my needed savings and leveled again. Turned it on today and same issue. Reinstalled firmware and it’s up and running. 

What can I do next?  I don’t want to set it up every time I use it. Thanks in advance!

**Comment by Cal-LF** [comment, marked as answer]
Try using a different SD card. If that does not fix it, the mainboard may be failing.
Another cause may be connector. Unplug and plugin all the connectors, even on the mainboard.

**Reply by immortallyinsane** [reply]
Thanks, it happens when no card is in but will try a different card and reformat it. I reset to defaults today and set it up again today. I have the CR touch on the way so I will double check all connections. 

One thing I may have changed was the “resume at power failure”. This wasn’t checked and I know it’s the only setting I changed to on recently.

**Reply by Cal-LF** [reply]
@immortallyinsane Resume after power failure feature is turned off on my setup. It uses the SD card intensively and may cause the card to fail, IMO.

**Reply by immortallyinsane** [reply]
Thanks for the input. Before powering off the printer I have been removing the SD card. So far no issues so the problem may be the card. I have another card showing up today and will update in a few days. Thanks to all that took time to read and give feedback.

**Comment by immortallyinsane** [comment]
As others suggested I am 99% sure this was the SD card. I have updated the firmware, reset to defaults and had no issues until I accidentally used the card again and powered on the printer. I cut that card in half and junked it!  Appreciate the time and suggestions!
